tests/functional/docs/__init__.py

```diff
@@ -53,48 +76,49 @@ def get_title_section_for(self, service_name):
 
     def get_method_document_block(self, operation_name, contents):
         contents = contents.decode('utf-8')
-        start_method_document = '  .. py:method:: %s(' % operation_name
+        start_method_document = '.. py:method:: %s(' % operation_name
         start_index = contents.find(start_method_document)
         self.assertNotEqual(start_index, -1, 'Method is not found in contents')
         contents = contents[start_index:]
-        end_index = contents.find(
-            '  .. py:method::', len(start_method_document)
-        )
+        end_index = contents.find('.. py:method::', len(start_method_document))
         contents = contents[:end_index]
         return contents.encode('utf-8')
 
     def get_parameter_document_block(self, param_name, contents):
         contents = contents.decode('utf-8')
-        start_param_document = '    :type %s:' % param_name
+        start_param_document = '  :type %s:' % param_name
         start_index = contents.find(start_param_document)
         self.assertNotEqual(start_index, -1, 'Param is not found in contents')
         contents = contents[start_index:]
-        end_index = contents.find('    :type', len(start_param_document))
+        end_index = contents.find('  :type', len(start_param_document))
```

Comment:
When client methods were all on one page, there was additional indentation since they all fell under the client class:
```
class S3.Client
  ...
  abort_multipart_upload(**kwargs)
  ...
```
Since they're now on their own page there is less indentation:
```
abort_multipart_upload(**kwargs)
  ...
```